Brewfile.lock.json

```diff
@@ -0,0 +1,155 @@
+{
```

Comment:
^ I'm no sure whether `Brewfile.lock.json` should be committed or not; we'll need to find out the hard way I guess...

Reply:
yeah not sure, feels right to commit